docs/src/docs/asciidoc/user/clustered-cache.adoc

```diff
@@ -0,0 +1,79 @@
+---
+---
+= Clustered Cache
+ifndef::sourcedir[]
+include::common.adoc[]
+endif::sourcedir[]
+
+ifdef::notBuildingForSite[]
+include::menu.adoc[]
+endif::notBuildingForSite[]
+
+== Introduction
+
+To enable Clustering with Terracotta, firstly you will have to start the http://terracotta.org[Terracotta server] configured with clustered storage.
+And then need a cache manager with clustering capabilities so that the caches can consume the clustered storage.
+
+== Starting Terracotta server
+
+You can either download the Terracotta 5.x distribution from http://terracotta.org/ or use the ehcache-clustered kit generated by the ehcache3 build.
```

Comment:
Why mention the distribution from terracotta.org? if you do that, it may lack the clustering libraries for Ehcache. Drop that part of the sentence. Only talk about the ehcache 3 kit which will be available ... you can remain vague on that point, it will be determined later.

Something like:
> We assume that you have the clustered Ehcache kit available locally.